packages/devtools/client/src/routes/overview/page.tsx

```diff
@@ -120,3 +120,9 @@ const Description = styled(Text)({
 const LogoHeading = styled.img({
   width: '10rem',
 });
+
+const LogoImage = styled.img({
```

Comment:
Why do we use emotion and CSS modules at the same time

Reply:
I perfer to use styled component from emotion, but I am not sure about is this will cause bundle size increase much. Especially for the mountpoint since there aren't many style files and it's more sensitive to bundle size. Maybe I will find a better styling solution and rewrite all of them later.

Reply:
Using CSS modules already meets our needs without introducing additional dependencies or runtime code.

Reply:
I initially used emotion based on my personal flavour, but I will reassess its necessity in the future.